src/Protocol/Protocol18x.cpp

```diff
@@ -1123,6 +1147,68 @@ void cProtocol180::SendDisplayObjective(const AString & a_Objective, cScoreboard
 
 
 
+void cProtocol180::SendSetSubTitle(const cCompositeChat & a_SubTitle)
+{
+	ASSERT(m_State == 3);  // In game mode?
+
+	cWorld * World = m_Client->GetPlayer()->GetWorld();
+	bool ShouldUseChatPrefixes = (World == nullptr) ? false : World->ShouldUseChatPrefixes();
+
+	cPacketizer Pkt(*this, 0x45);  // Title packet
+	Pkt.WriteVarInt32(1);  // Set subtitle
+
+	Pkt.WriteString(a_SubTitle.CreateJsonString(ShouldUseChatPrefixes));
```

Comment:
I think neither titles nor subtitles should ever have chat prefixes, or should they? @tigerw what do you think?
Also, how about just calling `SendSetRawSubTitle(a_SubTitle.CreateJsonString(false))`? That way the actual packet format is centralized to a single location and it's easier to clone+upgrade the protocol to a newer version.